lib/invoiced/operations/update.rb

```diff
@@ -8,7 +8,7 @@ def save(params = {})
                     update[k] = @values[k]
                 end
 
-                update.merge(params)
+                update = update.merge(params)
```

Comment:
Can also be done as `update.merge!(params)` but I prefer to use non-bang methods unless there's a very good reason not to.